.github/workflows/fcos-podman-next-build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build FCOS image with packages from rhcontainerbot/podman-next
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_NAME: fcos
+  IMAGE_TAGS: podman-next ${{ github.sha }}
+  IMAGE_REGISTRY: quay.io/podman
+  REGISTRY_USER: ${{ secrets.QUAY_PODMAN_USERNAME }}
+  REGISTRY_PASSWORD: ${{ secrets.QUAY.PODMAN_PASSWORD }}
+  COPR_OWNER: rhcontainerbot
+  COPR_PROJECT: podman-next
+
+jobs:
+  wait-for-task:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+
+    - name: Set up wait-for-copr
+      run: |
+        pip3 install git+https://github.com/packit/wait-for-copr.git@main
+
+    - name: Wait for successful podman-next build with the latest commit
+      run: |
+        wait-for-copr --owner ${{ env.COPR_OWNER }} --project ${{ env.COPR_PROJECT }} podman `git rev-parse --short ${{ github.sha }}`
+        echo "podman-next build successful."
+
+    - name: Build FCOS Image
+      id: build_image
+      # Ref: https://github.com/redhat-actions/buildah-build
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_NAME }}
+        tags: ${{env.IMAGE_TAGS }}
+        containerfiles: |
+          ./contrib/podman-next/fcos-podmanimage/Containerfile .
+
+    - name: Push to Quay
+      id: push-to-quay
+      # Ref: https://github.com/redhat-actions/push-to-registry
+      uses: redhat-actions/push-to-registry@v2
+      with:
+        image: ${{ steps.build-image.outputs.image }}
+        tags: ${{ steps.build-image.outputs.tags }}
+        registry: ${{ env.IMAGE_REGISTRY }}
+        username: ${{ env.REGISTRY_USER }}
+        password: ${{ env.REGISTRY_PASSWORD }}
```

Comment:
Minor: Consider eliminating the `env:` definitions for `registry`, `username`, and `password.  They're unnecessary since you can just reference the `${{ secrets.WHATEVER }}` directly here.  It also limits exposure of credentials by confining the usage scope. 
 It's very easy to accidentally leak env. vars values and github's output filtering may not always catch them all.

Reply:
changing in update.